server/routes/bookedtoday/arrivals/confirmAddedToRollController.ts

```diff
@@ -20,7 +20,7 @@ export default class ConfirmAddedToRollController {
       const prison = await this.prisonService.getPrison(activeCaseLoadId)
       clearImprisonmentStatus(res)
       clearSex(res)
-      return res.render('pages/confirmAddedToRoll', { data, prison, offenderNumber })
+      return res.render('pages/bookedtoday/arrivals/confirmAddedToRoll.njk', { data, prison, offenderNumber })
```

Comment:
some places we have the `njk` suffix and others we don't - its not necessary so we could remove it everywhere